### ISSUE TITLE
Cant build

### ISSUE THREAD

**Issue by bystepii** [yantz74/Ido_Redmi3_Kernel#1]
``` bash
user Ido_Redmi3_Kernel-ido-l-oss # ./build.sh 
 Cleaning previous build 
mkdir: cannot create directory ‘/home/user/Ido_Redmi3_Kernel-ido-l-oss/yantz/modules/pronto’: No such file or directory
 Make config (yantz_defconfig) 
  HOSTCC  scripts/basic/fixdep
  GEN     /home/user/Ido_Redmi3_Kernel-ido-l-oss/out/Makefile
  HOSTCC  scripts/kconfig/conf.o
  SHIPPED scripts/kconfig/zconf.tab.c
  SHIPPED scripts/kconfig/zconf.lex.c
  SHIPPED scripts/kconfig/zconf.hash.c
  HOSTCC  scripts/kconfig/zconf.tab.o
In file included from scripts/kconfig/zconf.tab.c:2503:0:
/home/user/Ido_Redmi3_Kernel-ido-l-oss/scripts/kconfig/menu.c: In function ‘get_symbol_str’:
/home/user/Ido_Redmi3_Kernel-ido-l-oss/scripts/kconfig/menu.c:567:18: warning: ‘jump’ may be used uninitialized in this function [-Wmaybe-uninitialized]
     jump->offset = r->len - 1;
                  ^
/home/user/Ido_Redmi3_Kernel-ido-l-oss/scripts/kconfig/menu.c:528:19: note: ‘jump’ was declared here
  struct jump_key *jump;
                   ^
  HOSTLD  scripts/kconfig/conf
drivers/input/touchscreen/Kconfig:2: syntax error
drivers/input/Kconfig:28: missing end statement for this entry
drivers/input/Kconfig:5: missing end statement for this entry
drivers/Kconfig:1: missing end statement for this entry
drivers/input/touchscreen/Kconfig:1: invalid statement
drivers/input/Kconfig:217: unexpected end statement
drivers/input/Kconfig:227: unexpected end statement
drivers/Kconfig:193: unexpected end statement
/home/user/Ido_Redmi3_Kernel-ido-l-oss/scripts/kconfig/Makefile:105: recipe for target 'yantz_defconfig' failed
make[2]: *** [yantz_defconfig] Error 1
/home/user/Ido_Redmi3_Kernel-ido-l-oss/Makefile:502: recipe for target 'yantz_defconfig' failed
make[1]: *** [yantz_defconfig] Error 2
Makefile:130: recipe for target 'sub-make' failed
make: *** [sub-make] Error 2
 Build kernel using 2 cores 
/home/user/Ido_Redmi3_Kernel-ido-l-oss/Makefile:793: *** multiple target patterns.  Stop.
Makefile:130: recipe for target 'sub-make' failed
make: *** [sub-make] Error 2
 Kernel Compilation failed! Fix the errors! 
user Ido_Redmi3_Kernel-ido-l-oss # 
```

I'm on LinuxMint 18 64bits but I also tried with Ubuntu 16.04 64bits and I get the same error.

**Comment by yantz74** [comment]
Sorry there was a derp. I committed but not uploaded yet. For quick fix, edit drivers/input/touchscreen/Kconfig
There is a leading 's' character which wasn't supposed to be there. Delete it and it should build

Yantz

From: ma3axaka007
Sent: Friday, August 19, 20:45
Subject: [yantz74/Ido_Redmi3_Kernel] Cant build (#1)
To: yantz74/Ido_Redmi3_Kernel

user Ido_Redmi3_Kernel-ido-l-oss # ./build.sh Cleaning previous build mkdir: cannot create directory '/home/user/Ido_Redmi3_Kernel-ido-l-oss/yantz/modules/pronto': No such file or directory Make config (yantz_defconfig) HOSTCC scripts/basic/fixdep GEN /home/user/Ido_Redmi3_Kernel-ido-l-oss/out/Makefile HOSTCC scripts/kconfig/conf.o SHIPPED scripts/kconfig/zconf.tab.c SHIPPED scripts/kconfig/zconf.lex.c SHIPPED scripts/kconfig/zconf.hash.c HOSTCC scripts/kconfig/zconf.tab.o In file included from scripts/kconfig/zconf.tab.c:2503:0: /home/user/Ido_Redmi3_Kernel-ido-l-oss/scripts/kconfig/menu.c: In function 'get_symbol_str': /home/user/Ido_Redmi3_Kernel-ido-l-oss/scripts/kconfig/menu.c:567:18: warning: 'jump' may be used uninitialized in this function [-Wmaybe-uninitialized] jump->offset = r->len - 1; ^ /home/user/Ido_Redmi3_Kernel-ido-l-oss/scripts/kconfig/menu.c:528:19: note: 'jump' was declared here struct jump_key _jump; ^ HOSTLD scripts/kconfig/conf drivers/input/touchscreen/Kconfig:2: syntax error drivers/input/Kconfig:28: missing end statement for this entry drivers/input/Kconfig:5: missing end statement for this entry drivers/Kconfig:1: missing end statement for this entry drivers/input/touchscreen/Kconfig:1: invalid statement drivers/input/Kconfig:217: unexpected end statement drivers/input/Kconfig:227: unexpected end statement drivers/Kconfig:193: unexpected end statement /home/user/Ido_Redmi3_Kernel-ido-l-oss/scripts/kconfig/Makefile:105: recipe for target 'yantz_defconfig' failed make[2]: *_\* [yantz_defconfig] Error 1 /home/user/Ido_Redmi3_Kernel-ido-l-oss/Makefile:502: recipe for target 'yantz_defconfig' failed make[1]: **\* [yantz_defconfig] Error 2 Makefile:130: recipe for target 'sub-make' failed make: **\* [sub-make] Error 2 Build kernel using 2 cores /home/user/Ido_Redmi3_Kernel-ido-l-oss/Makefile:793: **\* multiple target patterns. Stop. Makefile:130: recipe for target 'sub-make' failed make: **\* [sub-make] Error 2 Kernel Compilation failed! Fix the errors! user Ido_Redmi3_Kernel-ido-l-oss #

I'm on LinuxMint 18 64bits but I also tried with Ubuntu 16.04 64bits and I get the same error.

##

**Comment by yantz74** [comment]
fix @ 9ed9059
